### PR TITLE
WT-7288 Construct many-dhandle-wtperf workload

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
Initial many dhandle wtperf workload

### CODE REVIEW

File: bench/wtperf/runners/many-dhandle-stress.wtperf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# This workload uses 10000 tables and the workload is evenly distributed among them. 
+# Configured with idle_table_cycle where it creates, opens and drops a table and it
+# fails fail if that takes more than configured time.
```
Comment (sulabhM) [inline bench/wtperf/runners/many-dhandle-stress.wtperf:3]: "fails fail" --> "fails"

Comment (sulabhM) [reply bench/wtperf/runners/many-dhandle-stress.wtperf:3]: For your consideration, rewording as follows:
"The workload creates, opens and drop tables, and it fails if the time taken is more than the configured idle_table_cycle"

**Review by sulabhM — APPROVED** [review]
LGTM, with modifying the comment.

File: bench/wtperf/runners/many-dhandle-stress.wtperf (hunk 1 above)
Comment (raviprakashgiri29) [reply bench/wtperf/runners/many-dhandle-stress.wtperf]: Updated the comment section.

**Pull request merged**